stdlib/distutils/dist.pyi

```diff
@@ -60,7 +60,7 @@ class DistributionMetadata:
 class Distribution:
     cmdclass: dict[str, type[Command]]
     metadata: DistributionMetadata
-    def __init__(self, attrs: Mapping[str, Any] | None = None) -> None: ...
+    def __init__(self, attrs: MutableMapping[str, Incomplete] | None = None) -> None: ...
```

Comment:
This didn't really fit into any upcoming PR that I had so I bundled it here where I also updated setuptool's `Distribution`.

In both `attrs['license']` is assigned to and `attrs['licence']` is deleted (notice s vs c)